docs/reference/connectors/common_configuration.md

```diff
@@ -188,4 +188,23 @@ end;
 
 ### none
 
-No `Authorization` is used.
\ No newline at end of file
+No `Authorization` is used.
+
+## Socket Options
```

Comment:
since they are used only by TPC and UDP respecively should they live in the TCP/UDP doc instead of common socket options?